### PR TITLE
refactor: multiplayer focus

### PR DESCRIPTION
**Pull request by bfelbo**
Change multiplayer examples to be top-level and remove singleplayer code as it's being deprecated (see warning on https://developers.rune.ai/docs/singleplayer/intro).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ SDK for building multiplayer HTML5 games that run inside Rune on [Android](https
...
+| [Demo](https://developers.rune.ai/multiplayer/examples/tic-tac-toe/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/tic-tac-toe) | [Demo](https://developers.rune.ai/multiplayer/examples/outmatched/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/outmatched) | [Demo](https://developers.rune.ai/multiplayer/examples/tic-tac-toe/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/sudoku) | [Demo](https://developers.rune.ai/multiplayer/examples/pinpoint/), [Source](https://github.com/rune/rune-games-sdk/tree/staging/examples/pinpoint) |
```
Comment (Gongreg) [inline README.md]: Is the path here correct to the demos?

Comment (bfelbo) [reply README.md]: Yes, until we change the docs to no longer have singleplayer docs. We could consider also doing that now I guess 🤠 But I'd say let's do it separately as this is a big PR already and there's some choices to be made there.

Comment (bfelbo) [reply README.md]: Ah, nvm. You're right 👍

Comment (bfelbo) [reply README.md]: Fixed in https://github.com/rune/rune-games-sdk/pull/167/commits/277961757cb0cd1f3186c583eaca938cb817360b.

**Review by Gongreg — APPROVED** [review]

**Pull request merged**